src/components/side-navigation/side-navigation-item.component.html

```diff
@@ -0,0 +1,11 @@
+<div class="menu-item" (click)="select()">
+
+  <i class="hpe-icon menu-item-icon" *ngIf="icon" [ngClass]="icon"></i>
+  <span class="menu-item-title">{{ header }}</span>
+  <span class="menu-item-state-icon hpe-icon hpe-previous" [class.expanded]="expanded" *ngIf="children.length !== 1"></span>
```

Comment:
I think the chevron should be on the left of the nav item, as per MF guidelines. Although this differs from the Keppel theme, that is no longer our priority. This will make it easier to style.